package.json

```diff
@@ -1,5 +1,6 @@
 {
 	"dependencies": {
+		"@fontsource/goldman": "^4.5.9",
```

Comment:
Still need to actually use this. Remove the font files and put an import in the scss file

Reply:
```css
@import "@fontsource/goldman/latin.css";
```